scripts/install-dev.sh

```diff
@@ -154,48 +153,34 @@ install_slack_cli() {
         echo -e "\x1b[2m\n$slack_cli_url"
         curl -# -fLo "$slack_cli_install_dir/slack-cli.tar.gz" "$slack_cli_url"
         echo -e "\x1b[0m"
-        if [ "$SLACK_CLI_DEV_VERSION" == "dev"  ]; then
+        if [ "$SLACK_CLI_DEV_VERSION" == "dev" ]; then
                 delay 0.2 "💾 Successfully downloaded the latest build to $(home_path "$slack_cli_install_dir/slack-cli.tar.gz")"
         else
                 delay 0.2 "💾 Successfully downloaded Slack CLI v$LATEST_SLACK_CLI_VERSION to $(home_path "$slack_cli_install_dir/slack-cli.tar.gz")"
         fi
 
-        delay 0.3 "📦 Extracting the Slack CLI command binary to $(home_path $slack_cli_bin_path)"
+        delay 0.3 "📦 Extracting the Slack CLI command binary to $(home_path "$slack_cli_bin_path")"
         tar -xf "$slack_cli_install_dir/slack-cli.tar.gz" -C "$slack_cli_install_dir"
         chmod +x "$slack_cli_bin_path"
-        rm "$slack_cli_install_dir/slack-cli.tar.gz"
 
-        delay 0.1 "🔗 Adding a symbolic link from /usr/local/bin/$SLACK_CLI_NAME to $(home_path $slack_cli_bin_path)"
-        if [ ! -d /usr/local/bin ]; then
-            echo -e "⚠️  The /usr/local/bin directory does not exist!"
-            echo -e "🔐 Please create /usr/local/bin directory first and try again..."
-            return 1
-        fi
-        if [ -w /usr/local/bin ]; then
-                ln -sf "$slack_cli_bin_path" "/usr/local/bin/$SLACK_CLI_NAME"
-        else
-                echo -e "⚠️  Failed to create a symbolic link!"
-                delay 0.1 "🔖 The installer doesn't have write access to /usr/local/bin"
-                echo -e "🔐 Please check permission and try again..."
-                return 1
-        fi
+        delay 0.1 "📠 Removing packaged download files from $(home_path "$slack_cli_install_dir/slack-cli.tar.gz")"
+        rm "$slack_cli_install_dir/slack-cli.tar.gz"
 
-        if [ $(command -v $SLACK_CLI_NAME) ]; then
-                echo -e ""
-                echo -e "✨ Success! The Slack CLI (build: $SLACK_CLI_DEV_VERSION) is now installed!"
-                echo -e ""
-                sleep 0.4
+        if [ -d "/usr/local/bin" ] && [ -w "/usr/local/bin" ]; then
+                local_bin_path="/usr/local/bin"
         else
-                echo "📁 Manually add the Slack CLI command directory to your shell profile"
-                echo "   export PATH=\"$slack_cli_install_bin_dir:\$PATH\""
+                local_bin_path="$HOME/.local/bin"
+                mkdir -p "$local_bin_path"
         fi
+        delay 0.1 "🔗 Adding a symbolic link from $(home_path "$local_bin_path/$SLACK_CLI_NAME") to $(home_path "$slack_cli_bin_path")"
+        ln -sf "$slack_cli_bin_path" "$local_bin_path/$SLACK_CLI_NAME"
```

Comment:
🗣️ note: The big change of this PR moves output checks of a **configured** CLI to later but not this `install_slack_cli` function. This is to change if statements that follow-